tests/linearizability/linearizability_test.go

```diff
@@ -54,15 +58,22 @@ var (
 		clientCount: 12,
 		traffic:     DefaultTraffic,
 	}
+	AuthTrafficConfig = trafficConfig{
+		minimalQPS:  100,
+		maximalQPS:  200,
+		clientCount: 8,
+		traffic:     DefaultTrafficWithAuth,
+	}
 )
 
 func TestLinearizability(t *testing.T) {
 	testRunner.BeforeTest(t)
 	tcs := []struct {
-		name      string
-		failpoint Failpoint
-		config    e2e.EtcdProcessClusterConfig
-		traffic   *trafficConfig
+		name        string
+		failpoint   Failpoint
+		config      e2e.EtcdProcessClusterConfig
+		traffic     *trafficConfig
+		clientCount int
```

Comment:
This client count doesn't seem to be used. Please remove it and use `TrafficConfig.clientCount`